python/version_details/version_details.py

```diff
@@ -322,7 +322,8 @@ def note_threads(self):
         The currently loaded Note threads keyed by Note entity id and
         containing a list of Shotgun entity dictionaries.
 
-        Example structure containing a single Note entity:
+        Example structure containing a single Note entity::
```

Comment:
With the single colon it was throwing an error on sphinx build, I think because of the following lines being indented. The double colons triggers a code block I think. Made the error go away at least. :)